### PR TITLE
Remove bitcointalkLink from frontend

### PR DESCRIPTION
**Pull request by LucoLe**
#### Summary

Removes the bitcointalkLink since it's not used anymore.

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: app/src/pages/Detailed/DetailedGQL.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ export const projectBySlugGQL = gql`
       description,
       websiteLink,
       email,
-      bitcointalkLink,
```
Comment (valo) [inline app/src/pages/Detailed/DetailedGQL.js]: Is this used anywhere else in the frontend?

Comment (LucoLe) [reply app/src/pages/Detailed/DetailedGQL.js]: not that I can see. This is the only occurrence of `bitcointalkLink`

Comment (Partysun) [reply app/src/pages/Detailed/DetailedGQL.js]: Yep. Closed

Comment (Partysun) [reply app/src/pages/Detailed/DetailedGQL.js]: Thank you, @LucoLe

**Pull request merged**